Makefile

```diff
@@ -106,7 +106,7 @@ image: yamls
 
 scan: 
 	mkdir -p $(BUILD_ROOT)/node-feature
-	docker run -v $(BUILD_ROOT)/node-feature:/out -v /var/run/docker.sock:/var/run/docker.sock  aquasec/trivy image -s CRITICAL,HIGH -f table  --vuln-type library -o /out/library_vulnerabilities.json --exit-code 22 ${IMAGE_TAG}
+	docker run -v $(BUILD_ROOT)/node-feature:/out -v /var/run/docker.sock:/var/run/docker.sock  aquasec/trivy image -s CRITICAL,HIGH -f table  --vuln-type library -o /out/library_vulnerabilities.json --exit-code 0 ${IMAGE_TAG}
```

Comment:
A comment here to explain the purpose behind the change in exit code will help - we may want to reset it in future once all the vulns are resolved

Reply:
Yes, actually was seeing go package vulns with older version of gprc health probe, v4.15 and so had changed the exit code, but now with v0.4.17 we have no go package vulnerabilities, so have reversed this exit-code change in the new commit